modules/Router.js

```diff
@@ -19,6 +19,7 @@ class Router extends Component {
     history: object,
     children: routes,
     routes, // alias for children
+    RoutingContext: func.isRequired,
```

Comment:
Works for me - I've seen people do both for custom component classes depending on the context and don't have a personal preference.